:sparkles: Add new points and test new presentation (#4)
zenika-open-source/how-to-promote-my-open-source: README.md

> This cheat sheet summarize important steps you should follow to promote your open-source project in the best conditions.

List of availables languages:

- [English](./README.md)
- [Français](./README-fr.md)

---

> # Cheat sheet

Reviewer: Is this title relevant as we already mention in the description that it's a cheat sheet ?

Author: You're right, I didn't see that. However I think we should put a title for this section because I add Useful links section. Any idea ? Maybe "Steps" ?

Author: Not relevent since I'll delete Useful Links section.

## 1. 🎢 Preparation step

- 😎 Choose a cool name for your project

- 💅 Make your README pretty

> - 👨‍🏫 Provide a demo of your project

Reviewer: ```suggestion
- ✨ Provide a demo of your project
```

- 👌 Insall/Usage must be super easy

- 📘 Create a clean and well structured documentation

## 2. 📢 Spread the word about your hard works

> - ⭐ Talk about your project to your friends and colleagues to get your first stars before publishing to popular website

Reviewer: ```suggestion
- ⭐ Talk about your project to your friends and colleagues to get your first stars before publishing to social networks
```

    <details>

>     <summary>Why ?</summary>

Reviewer: What do you think about replacing "Why ?" by "Details", in some cases I think we'll need to give more explanations

Author: I'm ok with that.

    <p>

  > A minimum of stars make your project more trustable then a zero star project.

    </p>
    </details>

- ↗️ Share the project on social medias an specialized platforms

> - ❓ Make a publication to explain the why and how of the project

Reviewer: ```suggestion
-  📃 Write articles that mention your project
```

> - 🕐 Take care to choose the best time to post and share articles

Reviewer: I think this point should be placed just after `↗️ Share the project on social medias an specialized platforms` ?

Author: I hesistated to do that, but `❓ Make a publication to explain the why and how of the project` is also concerned by this point. An other solution it put this point before `↗️ Share the project on social medias an specialized platform` and `❓ Make a publication to explain the why and how of the project`. What do you think about that ?

> - 📘 Manage your communication to make the project be in trending pages

Reviewer: I think this point is not something the user can do. It's not something he can controls. What do you think?

Author: That's ok. But I would like write somewhere that trending page is something the reader need to aim.

## 3. 🤝 Keep your users

> - ✨ Make update time to time

Reviewer: ```suggestion
- 🆕Try to regularly release new versions of your project
```

- ❗ Do not let issues open without answer

> - 🙏 Do not hesitate to ask for help

Reviewer: ```suggestion
- 🙏 Invite users to contribute
```

> - 🤝 Reward contributors

Reviewer: ```suggestion
- 🏆 Reward contributors
```

- 💬 Open a community chat platform

- 🔙 Ask for feedback

---

# Useful links

> - [List of beautiful readmes](https://github.com/matiassingers/awesome-readme)

Reviewer: How about putting this link in the details section of "💅 Make your README pretty" ?

Author: We can try this.
